schema/ridl/_example/example1-golden.json

```diff
@@ -9,10 +9,20 @@
    "type": "uint32",
    "fields": [
     {
+     "comments": [
+      {
+       "Value": "comment"
```

Comment:
how about instead of having separate "Value" per line we just have a single "comments" field and when we parse will join adjacent comments with `"\n"` ? feels cleaner and easier to work with at parse time..? but lmk

Reply:
Done: https://github.com/webrpc/webrpc/pull/252/commits/0dd03e8a8481656b071b34590accfe36b6442ee9

I don't know why I was complicating my life. Thanks for the suggestion. 

now it's much easier to read in tests as well